core/field.js

```diff
@@ -260,11 +305,13 @@ Blockly.Field.prototype.init = function() {
     // Field has already been initialized once.
     return;
   }
-  this.fieldGroup_ = Blockly.utils.dom.createSvgElement('g', {}, null);
+  this.fieldGroup_ = /** @type {!SVGGElement} **/
+      (Blockly.utils.dom.createSvgElement('g', {}, null));
   if (!this.isVisible()) {
     this.fieldGroup_.style.display = 'none';
   }
-  this.sourceBlock_.getSvgRoot().appendChild(this.fieldGroup_);
+  var sourceBlockSvg = (/** @type {!Blockly.BlockSvg} **/ (this.sourceBlock_));
```

Comment:
You don't need to wrap this in parenthesis, removing them will make in more consistent with other places.